integral_view/views/system.py

```diff
@@ -0,0 +1,53 @@
+import django
+import django.template
+
+from integral_view.forms import system_date_time_form
+from integralstor_utils.date_time import update_date_time
+
+def input_date_time(request):
+    return_dict = {}
+    try:
+        form = system_date_time_form.date_time_form()
+        return_dict['form'] = form
+    except Exception, e:
+        return_dict['base_template'] = "view_date_time.html"
+        return_dict["page_title"] = 'Update system and hardware date and time'
+        return_dict["error"] = 'Error viewing date time'
+        return_dict["error_details"] = str(e)
+        return django.shortcuts.render_to_response("logged_in_error.html", return_dict, context_instance=django.template.context.RequestContext(request))
+    else:
```

Comment:
The template should be called update_system_date_time.html because it actually is used to update and not to view the date and time.